src/test/fixtures/coordinate-allowed-with-reason-version-catalog-null-version/settings.gradle

```diff
@@ -0,0 +1,10 @@
+dependencyResolutionManagement {
+  versionCatalogs {
+    "libs" {
+      library("exam", "com.example", "example").withoutVersion()
```

Comment:
`withoutVersion` requires a Maven BOM, otherwise java compile does not have a working gav.